anaconda_project/project_commands.py

```diff
@@ -408,6 +408,8 @@ def description(self):
             description = self._attributes.get('windows', None)
         if description is None:
             description = self._attributes.get('conda_app_entry', None)
+        if description is None:
+            description = getattr(self, 'command', None)
```

Comment:
Basically I'd try to have some way this is documented in the base class, if the base class is going to refer to it.

(merge when you want/need to!)